### PR TITLE
More texts and options customization

### PR DESCRIPTION
**Pull request by sglkc**
Users can now modify almost every presence texts, also more support for users with custom client.
Some codes may need optimizations, as I'm _not_ too familiar with python...

### CODE REVIEW

**Comment by hugolgst** [comment]
@anned20 Could you also review it?

**Review by anned20 — CHANGES_REQUESTED** [review]
Didn't test it yet, but the code looks good.

File: python/vimsence.py [hunk 1]
```diff
@@ -122,21 +123,38 @@ def update_presence():
...
-    editing_text = 'Editing a {} file'
-    if (vim.eval("exists('{}')".format("g:vimsence_editing_large_text")) == "1"):
-        editing_text = vim.eval("g:vimsence_editing_large_text")
+    filesize = get_filesize()
+    filesizeb = get_filesizeb()
+    fileline = get_fileline()
+
+    # Replace function for customization by user,
+    # not a good code but it's fast and does the job
+    def replace_text(string):
```
[the remark this answers is not in the document]
Comment (sglkc) [reply python/vimsence.py]: I've renamed `replace_text` function to `parse_tags`, this should solve the request

**Review by sglkc — COMMENTED** [review]
Renamed `replace_text` to `parse_tags`

**Review by anned20 — CHANGES_REQUESTED** [review]
There has been a change in code style in the master branch.

Please update your code to use that coding style and run the pre-commit hook, description on how to do that can be found in the README.md.

Please rebase and squash your commits too after doing this.

**Pull request closed without merging**